### ISSUE TITLE
wx on mac

### ISSUE THREAD

**Issue by hugohonorem** [bluenote10/PandasDataFrameGUI#1]
WX is very hard to get installed on mac el capitan, spent some trying to install it yesterday but failed multiple of times. 

However, still under development? have you considered moving everything to like jupyter/ipython instead? And adding transformation of variables? something like the filter column like "if userid == 669944 change strincolumn to B"

**Comment by hugohonorem** [comment]
http://davixx.fr/blog/2016/01/25/wxpython-on-os-x-el-capitan/

this actually work for mac, you have to repackage it in el capitan..

**Comment by bluenote10** [comment]
Yes, I'm still working on this and checking OS X support is one of the next steps.

Regarding moving to jupyter: No, this is not something I have intended for now, since I'm mainly using dfgui from within `IPython.embed()` for debugging. That's also why an actual modification of the data frame is not really required for my workflow.

**Comment by hugohonorem** [comment]
Cool cool, would be nice with modification of data though, I'll see if I can come up with something.
